lte/gateway/c/core/oai/oai_mme/oai_mme.c

```diff
@@ -139,6 +139,7 @@ int main(int argc, char* argv[]) {
   event_client_init();
 
   CHECK_INIT_RETURN(mme_app_init(&mme_config));
+  CHECK_INIT_RETURN(amf_app_init(&amf_config));
```

Comment:
this should only initialized if converged core flag is set.

Reply:
resolved, added enable_converged_core check before initialize amf app.